recipes/moonrepo-moon/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "moonrepo-moon" %}
+{% set version = "1.27.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/moonrepo/moon/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e2f88f4af9534092692a09c49f4dd6d8d96535df7134f4f5c66c0aee3df57673
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}              # [not win]
+    - {{ stdlib("c") }}                # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ stdlib("m2w64_c") }}          # [win]
+    - cargo-bundle-licenses
+    - openssl
```

Comment:
This should have been in host

Reply:
I'm not that good with conda-forge. Can I ask you to explain where should I move this code?

Reply:
This should be in the `host:` section of `requirements`

Reply:
I see. I double-checked and removed unnecessary deps